src/UnAuthorizedRoutes/Signup/Signup.jsx

```diff
@@ -1,10 +1,52 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import { useForm, FormProvider } from "react-hook-form";
+import Input from "components/Input/Input";
+import useSignup from "queries/useSignup";
+import useLogin from "queries/useLogin";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
+
+const inputs = ["username", "password"];
 
 const Signup = () => {
```

Comment:
It's almost same as login page, so... probably create share form component and switch login or signup depends on link?

Reply:
Created shared UnAuthorized component on #9 .